devtools/conda-envs/conda.yaml

```
@@ -1,10 +1,12 @@
 name: test
```

Nit (similar for other files as well): when reproducing issues locally I've sometimes maybe had clashes with files like these and existing environments named `test`
```suggestion
name: deployment-env
```

---

fixed in [8b4ebf3](https://github.com/openforcefield/openff-toolkit/pull/1353/commits/8b4ebf3b32cf6bd79e4ddc309fd59adf3ae01420)